interface-definitions/vpn_ipsec.xml.in

```diff
@@ -1109,6 +1109,54 @@
                           </completionHelp>
                         </properties>
                       </leafNode>
+                      <node name="local">
```

Comment:
There is already a `prefix` node unter a site-to-site peers tunnel node. Can this probably me merged into an identical definition which then can be moved to a common `xml.i` include file to not have duplicated CLI definitions?